uiautomationutilities/src/main/java/com/microsoft/identity/client/ui/automation/broker/BrokerCompanyPortal.java

```diff
@@ -173,7 +171,7 @@ public void handleFirstRun() {
     }
 
     public void enrollDevice(@NonNull final String username,
-                             @NonNull final String password, @NonNull final boolean isFederated) {
+                             @NonNull final String password, @NonNull final Boolean isFederated) {
```

Comment:
Why `Boolean` as opposed to `boolean`?

Reply:
I hve now reverted it to boolean and also removed NonNull annotation